nifi-registry/nifi-registry-core/nifi-registry-flow-diff/src/main/java/org/apache/nifi/registry/flow/diff/StandardFlowDifference.java

```diff
@@ -116,18 +116,18 @@ public boolean equals(final Object obj) {
 
         // If both flows have a component A with an instance identifier, the instance ID's must be the same.
         if (componentA != null && componentA.getInstanceIdentifier() != null && other.componentA != null && other.componentA.getInstanceIdentifier() != null
-            && !componentA.getInstanceIdentifier().equals(other.componentA.getInstanceIdentifier())) {
+                && !componentA.getInstanceIdentifier().equals(other.componentA.getInstanceIdentifier())) {
             return false;
         }
 
         // If both flows have a component B with an instance identifier, the instance ID's must be the same.
         if (componentB != null && componentB.getInstanceIdentifier() != null && other.componentB != null && other.componentB.getInstanceIdentifier() != null
-            && !componentB.getInstanceIdentifier().equals(other.componentB.getInstanceIdentifier())) {
+                && !componentB.getInstanceIdentifier().equals(other.componentB.getInstanceIdentifier())) {
             return false;
         }
 
         return Objects.equals(componentAId, otherComponentAId) && Objects.equals(componentBId, otherComponentBId)
-            && Objects.equals(description, other.description) && Objects.equals(type, other.type)
-            && Objects.equals(valueA, other.valueA) && Objects.equals(valueB, other.valueB);
+                && Objects.equals(description, other.description) && Objects.equals(type, other.type)
+                && Objects.equals(valueA, other.valueA) && Objects.equals(valueB, other.valueB);
     }
-}
+}
```

Comment:
This file also contains a number of formatting changes, please revert so that the changes are scoped to the `hashCode()` method. Thanks!